eslint-config-react-typescript/index.js

```diff
@@ -77,4 +77,12 @@ module.exports = {
     ],
     '@typescript-eslint/explicit-module-boundary-types': 'off'
   },
+  overrides: [
+    {
+      'files': ['**/*.spec.jsx'],
+      'rules': {
+        'react/jsx-filename-extension': ['off'],
+      }
+    }
+  ],
```

Comment:
Why do we add overrides instead of just adding this rule with the others?

Reply:
will it work if we put this block in rules?